### DISCUSSION TITLE
Deleting a Media record/model doesn't delete the associated files

### DISCUSSION THREAD

**Discussion by DanielvdSpoel** [spatie/laravel-medialibrary#2893] [Q&A]
We use the following code: 
`
        $data = Media::Find($media); //$media is the id of the media we wish to delete
        $data->delete();
`

I expect this to also remove all the files, so we don't end up with all sorts of files that are deleted. But they are still there? How can we fix this? I also can't find anything in the docs about deleting media

**Comment by freekmurze** [comment]
This test proves that it is working as intended: https://github.com/spatie/laravel-medialibrary/blob/6a355b289f7f65033057ea154a551127f1c8ef87/tests/Feature/Media/DeleteTest.php#L18

Feel free to submit a failing tests for us to look at.
